SystemInfo/Value/ComposerPackage.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * File containing the ComposerPackage class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo\Value;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+/**
+ * Value for information about a Composer package.
+ */
+class ComposerPackage extends ValueObject implements SystemInfo
+{
+    /**
+     * Version.
+     *
+     * Example: v2.7.10
+     *
+     * @var string
+     */
+    public $version;
```

Comment:
But @bdunogier has a point, there should be a separate property for stability flag, this is about not how stable the package is, but which stability channel we pull in new packages on.

_But_ I think you'll only find this in composer.lock, like this:
```
    "stability-flags": {
        "ezsystems/ezpublish-kernel": 20,
        "ezsystems/repository-forms": 20,
        "ezsystems/ezplatform-solr-search-engine": 20,
        "ezsystems/platform-ui-bundle": 20,
        "ezsystems/behatbundle": 20
    },
```

20 here refers to `@dev` as set in composer.json in master branch of ezplatform.